app/models/tenant.rb

```diff
@@ -0,0 +1,13 @@
+class Tenant < ApplicationRecord
+  VALID_WEBSITE_REGEX = /\A(www.)?[^_\W][-a-zA-Z0-9_]+\.+[-a-zA-Z0-9]+\z/i
+  VALID_PHONE_REGEX = /\A[+]?[\d\-.() ]+\z/
+
+  has_many :users
+  has_many :companies, through: :subscriptions
```

Comment:
through: :subscriptions ?? what is subscriptions?

Reply:
changed to through: :users